### PR TITLE
Calculator homework check

### PR DESCRIPTION
**Pull request by CarlosGuadamuz0**

### CODE REVIEW

File: src/main/java/calculator/Calculator.java [hunk 1]
```diff
@@ -20,22 +19,17 @@ public Calculator() {
...
-            case ADDITION: {
-                return addition();
-            }
-            case DIVISION: {
-                return division();
-            }
-            default: {
-                throw new IllegalStateException("Unknown enumeration value ");
-            }
+        char operationCharacter = getOperationCharacter();
```
Comment (BohdanKalep) [inline src/main/java/calculator/Calculator.java:22]: Please do not change logic to if/else. 
Switch works fine with enums

Comment (CarlosGuadamuz0) [reply src/main/java/calculator/Calculator.java:22]: It says something about constant expresion required. I try everything and It didnt work

Comment (BohdanKalep) [reply src/main/java/calculator/Calculator.java:22]: You need to change char type to enum type on the line 22

File: src/main/java/calculator/Calculator.java [hunk 2]
```diff
@@ -3,11 +3,10 @@
  */
 package calculator;
 
+
+import static calculator.Operator.*;
```
Comment (BohdanKalep) [inline src/main/java/calculator/Calculator.java:7]: please avoid using wildcards

**Comment by CarlosGuadamuz0** [comment]
I think I solved it

File: src/main/java/calculator/CalculatorRunner.java [hunk 3]
```diff
@@ -7,7 +7,11 @@
 import java.util.Scanner;
 import java.util.Set;
 
+import static calculator.Operator.*;
```
Comment (BohdanKalep) [inline src/main/java/calculator/CalculatorRunner.java:10]: Please remove wildcard

File: src/main/java/calculator/Calculator.java [hunk 4]
```diff
@@ -3,11 +3,9 @@
  */
 package calculator;
 
+
+
```
Comment (BohdanKalep) [inline src/main/java/calculator/Calculator.java:7]: remove 2 extra lines

**Review by BohdanKalep — APPROVED** [review]

**Pull request merged**